ios/versioned-react-native/ABI30_0_0/EXPermissions/ABI30_0_0EXPermissions/ABI30_0_0EXLocationRequester.m

```diff
@@ -77,9 +77,12 @@ - (void)requestPermissionsWithResolver:(ABI30_0_0EXPromiseResolveBlock)resolve r
     _locMgr = [[CLLocationManager alloc] init];
     _locMgr.delegate = self;
 
-    if ([[NSBundle mainBundle] objectForInfoDictionaryKey:@"NSLocationWhenInUseUsageDescription"] &&
-               [_locMgr respondsToSelector:@selector(requestWhenInUseAuthorization)]) {
-        [_locMgr requestWhenInUseAuthorization];
+    NSBundle *bundle = [NSBundle mainBundle];
+
+    if ([bundle objectForInfoDictionaryKey:@"NSLocationAlwaysUsageDescription"] && [_locMgr respondsToSelector:@selector(requestAlwaysAuthorization)]) {
+      [_locMgr requestAlwaysAuthorization];
```

Comment:
We may need to split this code into an optional module. We comment out a `requestAlwaysAuthorization` call in RN that is unused (never called) because Apple detects the call and then rejects apps during review.

Reply:
Reverted change in this file - it must have been something that git did when rebasing or something. Now we have better solution for that, which passes Apple's static code analysis - it is already shipped with SDK31 (see #2343).